common/src/configuration/utils.rs

```diff
@@ -80,13 +80,13 @@ pub fn default_config(bootstrap: &ConfigBootstrap) -> Config {
         default_subdir("wallet/console-wallet.dat", Some(&bootstrap.base_path)),
     )
     .unwrap();
-    cfg.set_default("wallet.base_node_query_timeout", 900).unwrap();
-    cfg.set_default("wallet.transaction_broadcast_monitoring_timeout", 600)
+    cfg.set_default("wallet.base_node_query_timeout", 60).unwrap();
+    cfg.set_default("wallet.transaction_broadcast_monitoring_timeout", 60)
```

Comment:
These values should be in line with what is documented in the 2x `*.toml` files, or vice versa:
```
# This is the timeout period that will be used to monitor TXO queries to the base node (default = 30). Larger values
# are needed for wallets with many (>1000) TXOs to be validated.
base_node_query_timeout = 120
# This is the timeout period that will be used for base node broadcast monitoring tasks (default = 30)
#transaction_broadcast_monitoring_timeout = 30
# This is the timeout period that will be used for chain monitoring tasks (default = 30)
#transaction_chain_monitoring_timeout = 30
# This is the timeout period that will be used for sending transactions directly (default = 20)
#transaction_direct_send_timeout = 20
# This is the timeout period that will be used for sending transactions via broadcast mode (default = 30)
#transaction_broadcast_send_timeout = 30
```
Please align the `*.toml` files with the config defaults.